### PR TITLE
[blame] Remove redundant commit hash from results

### PR DESCRIPTION
**Pull request by juanjux**
Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

 
 
 - [X] I updated the documentation explaining the new behavior if any.
 - [X] I updated CHANGELOG.md file adding the new feature or bug fix.
 - [X] I updated go-mysql-server using `make upgrade` command if applicable.
 - [X] I added or updated examples if applicable.
 - [X] I checked that changes on schema are reflected into the documentation, if applicable.

### CODE REVIEW

**Comment by juanjux** [comment]
@ajnavarro done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Added BLAME function.
+- Removed redundant commit information from BLAME results.
```
Comment (ajnavarro) [inline CHANGELOG.md]: Please, use the format from https://keepachangelog.com/en/1.0.0/ .

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Added BLAME function.
```
Comment (ajnavarro) [inline CHANGELOG.md]:
```suggestion
## Added
- Added BLAME function.
```

File: CHANGELOG.md (hunk 1 above)
Comment (ajnavarro) [inline CHANGELOG.md]:
```suggestion
## Fixed
- Removed redundant commit information from BLAME results.
```

Comment (juanjux) [reply CHANGELOG.md]: :+1:

**Comment by carlosms** [comment]
Why are there changes in `go.sum`, if there are no changes in the imports?

**Comment by juanjux** [comment]
@carlosms `make upgrade` I guess.

**Comment by juanjux** [comment]
Dont merge yet, trying to sort the go.mod / go.sum wtfuckery.

**Comment by juanjux** [comment]
Typical goland go.mod/go.sum "just because" update fixed, shell we merge @ajnavarro?

**Pull request merged**